### PR TITLE
Remove MANIFEST file when making the 'clean' target

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,6 +8,7 @@ all:
 clean:
 	find . -name "*.so" -o -name "*.pyc" -o -name "*.md5" -o -name "*.pyd" | xargs rm -f
 	find . -name "*.pyx" -exec ./tools/rm_pyx_c_file.sh {} \;
+	rm -f MANIFEST
```
Comment (soupault) [inline Makefile]: Where do you get this file from?

Comment (stefanv) [reply Makefile]: When you do source builds, a `MANIFEST` file is generated from `MANIFEST.in`.

Comment (soupault) [reply Makefile]: It is actually created during `python setup.py sdist` run.

File: tools/rm_pyx_c_file.sh [hunk 2]
```diff
@@ -4,3 +4,8 @@ filename="${1%.*}".c
 if [ -e "$filename" ]; then
 	rm -f "$filename";
 fi
+
+filename="${1%.*}".cpp
```
Comment (soupault) [inline tools/rm_pyx_c_file.sh]: I'd rename the script then.

File: Makefile (hunk 1 above)
Comment (stefanv) [reply Makefile]: How is that not a source build? :)

File: tools/rm_pyx_c_file.sh (hunk 2 above)
Comment (stefanv) [reply tools/rm_pyx_c_file.sh]: Sure, let's do that in another PR, if you want.

File: Makefile (hunk 1 above)
Comment (soupault) [reply Makefile]: For me, `python setup.py build_ext` or `pip install -e .` are source builds. `sdist` is `sdist`. :) Anyways...

**Pull request merged**